scripts/common/smoketest.go

```diff
@@ -0,0 +1,155 @@
+package common
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path"
+	"path/filepath"
+	"strconv"
+	"syscall"
+)
+
+type SmokeTest []*exec.Cmd
```

Comment:
Now that I think about it, smoke test is already used in the scootapi binary and this is kinda the same, but kinda different:

You can run a scootapi smoke test against aurora scheduler/workers (or against local scheduler and aurora workers).

WDYT of giving this a different name to emphasize that? E.g., "swarm test", because this creates a swarm.